.gitignore

```
@@ -0,0 +1 @@
+D:\git\temperatury\.vs
```

```suggestion
.vs
```

is enough. Using absolute path in `.gitignore` is wrong.
You need to treat this repository like it is using by several developers.

What if i keep it on `C://`?
What is i download this project on macbook?